src/codeu/chat/client/core/Admin.java

```diff
@@ -0,0 +1,137 @@
+package codeu.chat.client.core;
+
+import codeu.chat.util.PasswordStorage;
+import codeu.chat.util.Uuid;
+
+import java.io.Console;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.Map;
+import java.util.Set;
+
+/**
+ * Class for admin-related tasks, such as managing admins and checking passwords
+ */
+public class Admin {
+
+  private Set<Uuid> admins, noPasswords;
```

Comment:
nit: new line for each declaration:
`private Set<Uuid> admins;
  private Set<Uuid>  noPasswords;
 `